### PR TITLE
Create anshsaxena1703.json

### PR DESCRIPTION
**Pull request by anshsaxena1703**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/anshsaxena1703.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "socials": [
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/anshsaxena1703"
+    },
+    {
+      "icon": "FaLinkedin",
+      "url": "https://www.linkedin.com/in/anshsaxena1703/"
+    }
+  ]
```
Comment (sital002) [inline data/anshsaxena1703.json:19]:
```suggestion
  ],
```
There is a traiking comma missing

File: data/anshsaxena1703.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Ansh Saxena",
+  "bio": "Open Source Developer | Full Stack Developer | Video Editor"
+  "tags" : [
+    "Open Source",
+    "Web Development",
+    "Cloud Computing",
+    "Machine Learning",
+  ]
```
Comment (sital002) [inline data/anshsaxena1703.json:9]:
```suggestion
  ],
```
Please add comma here

File: data/anshsaxena1703.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Ansh Saxena",
+  "bio": "Open Source Developer | Full Stack Developer | Video Editor"
```
Comment (sital002) [inline data/anshsaxena1703.json:3]:
```suggestion
  "bio": "Open Source Developer | Full Stack Developer | Video Editor",
```

File: data/anshsaxena1703.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  "tags" : [
+    "Open Source",
+    "Web Development",
+    "Cloud Computing",
+    "Machine Learning",
```
Comment (sital002) [inline data/anshsaxena1703.json:8]:
```suggestion
    "Machine Learning"
```

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged

**Review by SaraJaoude — APPROVED** [review]
I have committed the changes for the JSON to pass proposed by one of the maintainers and I will now merge this PR.

**Pull request merged**